projects/plugins/jetpack/modules/custom-css.php

```diff
@@ -18,24 +18,14 @@
 function jetpack_load_custom_css() {
 	// If WordPress has the core version of Custom CSS, load our new version.
 	// @see https://core.trac.wordpress.org/changeset/38829
+	// @todo When should we remove the migration?
```

Comment:
This was for people upgrading from WordPress before 4.7, so 6 years ago. I vote for "now".

Hmm... If we do that, then is custom-css.php ever used at all anymore? In that case we could do #25182 by deleting it here and letting wpcom own it.

Reply:
Looping in @kraftbj as the creator of the todo [here](https://github.com/Automattic/jetpack/blob/e8fb66c1095c441b4dde3d7308e41c79221f46bb/projects/plugins/jetpack/modules/custom-css.php) too.

I think it's a good idea. I don't want to hold this PR up while I make sure I fully understand everything involved, but looking at it just now my understanding is that we'd be removing `custom-css/custom-css.php` and `custom-css/migrate-to-core.php`, and editing the `jetpack_load_custom_css` function in `modules/custom-css.php` to be something like this:

```
function jetpack_load_custom_css() {
	// If WordPress has the core version of Custom CSS, load our new version.
	// @see https://core.trac.wordpress.org/changeset/38829
	if ( function_exists( 'wp_get_custom_css' ) ) {
		if ( ! function_exists( 'wp_update_custom_css_post' ) ) {
			wp_die( 'Please run a SVN up to get the latest version of trunk, or update to at least 4.7 RC1' );
		}
		include_once __DIR__ . '/custom-css/custom-css/preprocessors.php';
		include_once __DIR__ . '/custom-css/custom-css-4.7.php';
		return;
	}
	wp_die( 'Please update to at least WordPress 4.7 to be able to make use of the Jetpack Custom CSS feature' );
}
```
If so, a very quick test seems to indicate there are no issues with current JP / WP versions but perhaps a new PR will allow us to test / refine without holding this PR up.

Reply:
For the record, doing that in a separate PR is fine with me.

Reply:
As for the proposed code snippet, since we don't support WP < 6.0 now there's probably no need to check for 4.7. So I'd just do like
```
function jetpack_load_custom_css() {
	include_once __DIR__ . '/custom-css/custom-css/preprocessors.php';
	include_once __DIR__ . '/custom-css/custom-css-4.7.php';
}
```
Maybe even rename `custom-css-4.7.php` to remove the version number too.